app/assets/stylesheets/chat.scss

```diff
@@ -1198,7 +1198,50 @@
     width: 10%;
   }
 }
+.connect-draw {
+  margin-top: 30px;
+  .draw-title {
+    display: flex;
+    justify-content: flex-start;
+  }
+  .colors {
```

Comment:
Follow [BEM](http://getbem.com/naming/) naming conventions.

e.g. `.connect-draw__colors` instead of `.connect-draw`.